test/CommonDatabaseTest.hx

```diff
@@ -32,7 +32,7 @@ class CommonDatabaseTest extends utest.Test {
 			var quoteField = @:privateAccess cls.manager.quoteField;
 			var name = cls.manager.dbInfos().name;
 			try {
-				Manager.cnx.request('DROP TABLE ${quoteField(name)}');
+				Manager.cnx.request('DROP TABLE IF EXISTS ${quoteField(name)}');
```

Comment:
There isn't a major problem in doing this, but *why is it necessary?*

SQL errors due to inexistent tables should already be cleanly swallowed by the `catch` bellow.

Reply:
It's stop useless exception raises which makes my annoy at debugging code on hl target

Reply:
Fair enough, but then the try-catch should be removed as well.